src/views/MapView.vue

```diff
@@ -25,22 +25,15 @@
           v-bind:setBlocks="setCompleteStreets"
           v-bind:title="'Complete'"/>
     </div>
-    <div class="header-bar">
-      <!-- <b-button v-if="reservedFilter === 0" disabled>Available blocks</b-button> -->
-      <!-- <b-button v-if="reservedFilter === 0" disabled>Blocks near me</b-button> -->
-      <h3 v-if="reservedFilter === 1 && !!activeStreetId">Block {{this.activeStreetId}}</h3>
-      <b-button v-on:click="labelsVisible = !labelsVisible">
-        {{`${(this.labelsVisible ? "Hide" : "Show")} block labels`}}
-      </b-button>
-    </div>
+
+    <hr />
```

Comment:
what is this? can we give it a more descriptive name?

Reply:
It's a horizontal rule, it's a built in element. It's that line between the header and the map